packages/styled-ui-docs/pages/scrollbar.mdx

```diff
@@ -68,6 +68,98 @@ Use the `disabled` attribute, the overflow is clipped, and the rest of the conte
 </Scrollbar>
 ```
 
+### The minimal thumb size
+
+Use the `minThumbSize` attribute to set the minimal size (in pixels) for thumbs.
+
+```jsx
+function MinThumbSizeScroll() {
+  const [minThumbSize, setMinThumbSize] = React.useState('32');
+  const [dataRow, setDataRow] = React.useState('8');
```

Comment:
Can `minThumbSize` and `dataRow` change to numeric values?

Reply:
@tinaClin fixed in  [2d67361]